project.clj

```diff
@@ -2,9 +2,31 @@
   :description "Spying library for testing"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
+
   :dependencies []
-  :plugins [[lein-cloverage "1.2.2"]]
-  :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.2"]]}}
+
+  :plugins [[lein-cloverage "1.2.2"]
+            [jonase/eastwood "0.3.14" :exclusions [org.clojure/clojure]]]
+
+  :eastwood {:exclude-linters
+             [ ;; clj-kondo will catch the wrong-arity cases
```

Comment:
```suggestion
             [;; clj-kondo will catch the wrong-arity cases
```
shakes fist at emacs autoindent

Reply:
annoyed at this one b/c it had me sus/questioning the right indent